.github/workflows/test-go-cli.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
         go-version: 1.17
 
     - name: Run Go CLI unit tests
-      run: cd ./cli && go test . 
+      run: cd ./cli && go test -v .
```

Comment:
I'm for not running with the verbose flag. On fail of one of the test cases, it's difficult to spot which case failed because we have so many cases. But when a test case does fail and we don't use the verbose flag, on the case that failed will come up, making it way easier to debug.

Reply:
Awesome. I'll remove it